templates/patterns/date_block/date_block.ui_patterns.yml

```diff
@@ -0,0 +1,25 @@
+date_block:
+  label: "Date block"
+  description: "Date block component"
+  variants:
+    default:
+      label: "Default event"
+      description: "Default event"
+    ongoing:
+      label: "Ongoing event"
+      description: "Ongoing event"
+    cancelled:
+      label: "Cancelled event"
+      description: "Cancelled event"
+    past:
+      label: "Past event"
+      description: "Past event"
+  fields:
+    date:
+      type: "DateValueObject"
+      label: "Date"
+      description: "A date value object."
+      preview:
+        day: "24"
+        month: "09"
+        year: "1981"
```

Comment:
we should add the parameter `weekday: "Thu."`

Reply:
Fixed.